src/react/Routes.jsx

```diff
@@ -45,18 +46,20 @@ function Routes() {
                     ]}
                 />
             </NavbarContainer>
-            <Switch>
-                <Route exact path="/" render={() => <Redirect to="/accounts" />}/>
+            <L.Container>
```

Comment:
What is this `L.Container` layer element used for?
We might want to remove it.
If needed, `ui-elements/ListingLayout2` components are specific to listing page, please use `ui-elements/Container` instead.

Reply:
Fixed, I found a way to remove L.Container without breaking changes.